src/DAAP2SQL.g

```diff
@@ -273,7 +273,7 @@ expr	returns [ pANTLR3_STRING result, int valid ]
 			/* String field: escape string, check for '*' */
 			else
 			{
-				if (op != ':')
+				if (op != ':' && op != '@')
```

Comment:
Can you add a comment describing why '@' is included? Paste a shortened version of the request from Apple Music, and also document the version number of Apple Music

Reply:
Did you want the comments duplicated wherever we added '@'? I pushed an update that just adds them above line 276 here.

Reply:
Fine to just have it here. Can you add the actual query? It is for future maintainers who might want to reproduce to see if the issue still exists.